pageserver/src/tenant/mgr.rs

```diff
@@ -245,7 +245,14 @@ pub async fn update_tenant_config(
     get_tenant(tenant_id, true)
         .await?
         .update_tenant_config(tenant_conf);
-    Tenant::persist_tenant_config(&conf.tenant_config_path(tenant_id), tenant_conf, false)?;
+    let tenant_config_path = conf.tenant_config_path(tenant_id);
+    Tenant::persist_tenant_config(&tenant_config_path, tenant_conf, false).with_context(|| {
+        format!(
+            "Failed to write tenant {} config to {}",
```

Comment:
NIT: note about the context message.

Reply:
Note this is the same as in tenant creation, and is a top level item... What should this be?

Reply:
Oh.

We could just remove the "failed to" prefix or leave as is, if you prefer similarity here.

Reply:
Aah ok. Well, I guess I prefer the similarity, perhaps it's easier picked up on the weekly  refactoring :)

Reply:
On a second thought, why not fix them both.

Reply:
Please see: [435d12d](https://github.com/neondatabase/neon/pull/3446/commits/435d12d912173b4451faf5b22638b4d0e112efc2) and [4f63e93](https://github.com/neondatabase/neon/pull/3446/commits/4f63e930b5e101bf5a4e98a85eccf82d6b0e6ad8)

Reply:
Thank you.